charts/newrelic-k8s-metrics-adapter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 description: A Helm chart to deploy the New Relic Kubernetes Metrics Adapter.
 name: newrelic-k8s-metrics-adapter
 version: 0.4.0
-appVersion: 0.2.0
+appVersion: 0.3.0
```

Comment:
how the version and appVersion will be managed when releasing?

Version should be manually updated and merged before releasing the app/chart or both at the same time ?